packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/sqm-tax-document-step-1.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+Feature: Tax Form Step One
+
+
+	Background: A user is prompted to enter their personal and initial tax information
+		Given a user is on the First step of the tax form
+
+
```

Comment:
Things to think about:
- A spec that describes what the country options are
- A spec that describes what the currency options are
- More on the hooks side, but what is a valid email - what validation are we using here

Things that we talked about that are not fully figured out but shouldn't be forgotten:
- what does this form look like after someone has gone through it for the first time then comes back
- What does the loading state for this form look like